### ISSUE TITLE
onMentioned doesnt return the original object

### ISSUE THREAD

**Issue by Alex61NN5** [ladderio/ngx-tribute#15]
Hey There,

When you select an item from the tribute list you only get a string value returned

```
onMentioned = new EventEmitter<string>();
```

now this does work when you are working with simple objects such as

```
{ key: 'baz', value: 'Baz' }
```

but in more advanced cases an object may look more like this

```
{
    name: 'John Smith',
    avatar: 'https://linktoavatar.com/picture.jpg',
    id: 32,
    email: 'johnsmith@gmail.com'
}
```

now say I select my user 'John Smith' - onMentioned only returns `@John Smith`

I suggest returning the string and the original object the `addEventListener()` function can be changed to the following

```
export class MentionObject {
   string: string,
   original: any
}

//...

@Output()
onMentioned = new EventEmitter<MentionObject>();

this.element.nativeElement.addEventListener('tribute-replaced', (event) => {
    const value = ['INPUT', 'TEXTAREA'].includes(
         this.element.nativeElement.tagName
    )
         ? this.element.nativeElement.value
          : this.element.nativeElement.innerHTML;

         this.onMentioned.emit({
             string: value,
             original: event.detail.item.original
         });

         if (this.control) {
             this.control.setValue(value);
         }
});
``` 

now the onMentioned will return 

```
{
    string: '@John Smith',
    original: {
        name: 'John Smith',
        avatar: 'https://linktoavatar.com/picture.jpg',
        id: 32,
        email: 'johnsmith@gmail.com'
    }
}
```

now a user could use any of the values from original item.

**Comment by agarbund** [comment]
Hey, thanks for the suggestion and sorry for the late response. I wasn't even aware that you can get an original item :)

Definitely could be useful but I wouldn't like to change existing API and make a major release. I'd rather add additional `@Output()` eg. `mentionItemSelected`.

Not sure if I'll have time to work on it this week. PR welcomed if you'd have time :)

**Comment by agarbund** [comment]
Published as `1.4.0`
